src/tests/system/tests/test_access_control.py

```diff
@@ -0,0 +1,139 @@
+"""
+SSSD Authentication Test Cases
+
+:requirement: access control
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.parametrize("method", ["su", "ssh"])
```

Comment:
Not sure it's worth running both with 'su' and 'ssh'. Maybe one of tests only..?

Reply:
I'll defer it to @danlavu

Reply:
su doesn't make sense because they are already logged into the host, and they are other mechanisms of more granular control for sudo.

Reply:
I kept ssh

Reply:
You can remove this line, and change the  method in the test to 

```
client.auth.ssh.password("user1", "Secret123"), "User login!"
```